s3roundtrip/s3roundtrip.go

```diff
@@ -73,7 +79,7 @@ func NewCollector(conf map[string]string) *collector {
 	}
 
 	// Pre-generate random data for object
-	data, err := rnd(1024 * 1024 * 10)
+	data, err := rnd(1024 * 1024 * 5 + 2)
```

Comment:
Make this configurable

Reply:
Also I don't see the point of generating MB of random data for this collector. This add some CPU load, just send 0x00.

Reply:
Size is now configurable via the `size` parameter in configuration file (default 5MiB + 2 bytes)